### PR TITLE
@craigspaeth -> add optional offset from bottom of page

### PR DESCRIPTION
**Pull request by 1aurabrown**

### CODE REVIEW

**Comment by craigspaeth** [comment]
:clap: well done!

File: README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ Callback when you reach the bottom of the page and ajax requests finish. Useful
 $.onInfiniteScroll(function() {
     // Fetch & append some content
 });
+
+$.onInfiniteScroll(function() {
+    // Pass an optional offset from the page bottom at which to trigger the callback.
+}, 100);
```
Comment (craigspaeth) [inline README.md]: I would actually prefer this be an options object so it has room to expand.

````
$.onInfiniteScroll(function() {
  // Pass an optional offset from the page bottom at which to trigger the callback.
}, { offset: 100 });
````

**Comment by craigspaeth** [comment]
:+1: This looks great, but plz address the options object comment first and happy to merge.

File: README.md (hunk 1 above)
Comment (1aurabrown) [inline README.md]: kk

**Comment by 1aurabrown** [comment]
closing, there are typos. will reopen.

**Comment by craigspaeth** [comment]
Solid! Thank yoU!

**Comment by 1aurabrown** [comment]
do i need to do a thing where i release a new version for npm?

**Comment by craigspaeth** [comment]
I need to give you npm publish rights to the project, but you should just point it to the git url instead, e.g. https://github.com/artsy/force/blob/master/package.json#L19

**Comment by craigspaeth** [comment]
@1aurabrown I also just published it, but it's more convenient for libraries that we just use internally to point to the artsy repo... I don't think anyone uses this plugin besides us

**Pull request merged**